src/geometry/poly3/measureBoundingSphere.js

```diff
@@ -0,0 +1,15 @@
+const vec3 = require('../../math/vec3')
```

Comment:
I am starting to think that we could have some generic measurement functions ?
might be premature, but could we not just operate on the underlying points ?

Reply:
nice idea. it would be possible now because all the geometries are point based. we will see.